src/SolidClient.js

```diff
@@ -136,8 +136,7 @@ class SolidClient {
     // Extract the password form's hidden fields
     const loginParams = { loginUrl };
     let match, inputRegex = /<input.*?name="([^"]+)".*?value="([^"]+)"/g;
-    while (match = inputRegex.exec(passwordForm))
-      loginParams[match[1]] = match[2];
+    while ((match = inputRegex.exec(passwordForm))) loginParams[match[1]] = match[2];
```

Comment:
Why does this need to be on the same line?

Reply:
ok